packages/build/config/tslint.common.json

```diff
@@ -22,7 +22,6 @@
     "no-unused-expression": true,
     "no-unused-variable": true,
     "no-var-keyword": true,
-    "triple-equals": [true, "allow-null-check", "allow-undefined-check"],
-    "typeof-compare": true
```

Comment:
Could you please explain why you are removing `typeof-compare`? I am sure you have good reasons, I'd just like to understand them too 😄

Reply:
Running the upgraded version of `tslint` complains that `typeof-compare` rule is now the default since TS 2.2 and it's not needed.

Reply:
^^^ It even prints that warning in our CI after tests pass (Travis & AppVeyor)